sys/libraries/deployment_settings.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class deployment_settings extends Library {
+
+    // Uncomment the next line to deactivate Missing Person Module
+    // private $missing_person = false;
```

Comment:
Default values anyway we are checking and setting in the functions, right? The only difference this make is the line 11 will be changed to `if (isset($deployment_settings[$var_name]))`.